src/const.ts

```diff
@@ -245,9 +245,7 @@ export const COW_SDK_DEV = [Network.MAINNET, Network.RINKEBY, Network.GNOSIS_CHA
   Record<number, CowSdk<SupportedChainId> | null>
 >((acc, networkId) => {
   try {
-    acc[networkId] = new CowSdk(networkId, {
-      isDevEnvironment: true,
-    })
+    acc[networkId] = new CowSdk(networkId)
```

Comment:
Any reason for changing this?

I haven't checked it throughly, but I suspect it might be the cause of duplicated orders, like seen [in this batch](https://pr168--explorer.review.gnosisdev.com/tx/0x5c31c2b6d7ea91d67bb1c2baa589b5b0e861461896e20928b746056b3380451c/?tab=orders)

I would avoid touching this if possible and leave that up to Ramiro's/mine PR which is adding Goerli support and also updating the SDK that does this
![Screen Shot 2022-08-18 at 11 29 38](https://user-images.githubusercontent.com/43217/185374311-81a170fe-1eec-4fb9-a08b-c3ddba4bbc1b.png)

Reply:
I think this broke when I merged from develop. Will fix this